fhir-install/src/main/docker/ibm-fhir-schematool/run.sh

```diff
@@ -242,49 +242,44 @@ function allocate_tenant {
 
         # Tenant Key
         TK_FILE_STANZA=""
+        TK_FILE='/opt/schematool/workarea/persistence.key'
         if [ ! -z "${TENANT_KEY}" ]
         then
-            TK_FILE='/opt/schematool/workarea/persistence.key'
             echo "${TENANT_KEY}" > ${TK_FILE}
-            TK_FILE_STANZA="--tenant-key-file ${TK_FILE}"
-
-            # Only in the case where we have a file we are loading we hit the issue
-            # where we need to override the default behavior or exiting on failure.
-            set +o errexit
-            set +o pipefail
+        else
+            # not specified by the client, so we're randomly generating and inlining it
+            TENANT_KEY=$(openssl rand -base64 32)
+            echo "TENANT_KEY: ${TENANT_KEY}"
+            echo "${TENANT_KEY}" > ${TK_FILE}
+            echo "tenant.key=${TENANT_KEY}" >> ${TOOL_INPUT_FILE}
         fi
 
+        # Only in the case where we have a file we are loading we hit the issue
+        # where we need to override the default behavior or exiting on failure.
+        set +o errexit
+        set +o pipefail
+
         SCHEMA_FHIR=$(get_property schema.name.fhir .persistence[0].schema.fhir)
         if [ -z "${SCHEMA_FHIR}" ] || [ "null" = "${SCHEMA_FHIR}" ]
         then
             SCHEMA_FHIR="FHIRDATA"
         fi
-        _call_db2 "--schema-name ${SCHEMA_FHIR} --allocate-tenant ${TENANT_NAME} ${TK_FILE_STANZA} --pool-size 1"
 
-        # Always reset
-        set -o errexit
-        set -o pipefail
+        _call_db2 "--schema-name ${SCHEMA_FHIR} --allocate-tenant ${TENANT_NAME} --tenant-key-file ${TK_FILE} --pool-size 5"
```

Comment:
I assume we'd done enough testing with a pool-size of 5 to feel good about this?

Reply:
Yeah - I'm pretty confident. at least I haven't hit issues with size 5 yet